### PR TITLE
Add TicketDiscussion

### PR DESCRIPTION
**Pull request by Elyas-Amini**
### Checklist:

- [ ] It refers to an [Issue](https://github.com/EasyMicroservices/FileManager/issues).
- [ ] It fixes a bug.
- [x It's a new feature (non-breaking change which adds functionality)
- [ ] It's a breaking (fix or feature that would cause existing functionality to change)
- [ ] My code follows the code style of this project.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] CHANGELOG is updated.
- [ ] Related language label is selected.

### CODE REVIEW

**Review by MahdiyarGHD — APPROVED** [review]

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]
Please remove controller logic

**Review by AmirmohammadNemati — APPROVED** [review]

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Contexts/SupportContext.cs [hunk 1]
```diff
@@ -59,6 +59,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 model.HasOne(bc => bc.Department)
                 .WithMany(b => b.TicketDepartments)
                 .HasForeignKey(bc => bc.DepartmentId);
+
+                model.HasIndex(bc => bc.TicketId).IsUnique();
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Contexts/SupportContext.cs]: Please remove IsUnique.

File: src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Contexts/SupportContext.cs [hunk 2]
```diff
@@ -75,6 +78,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
                 model.HasOne(bc => bc.Ticket)
                 .WithMany(b => b.TicketAssigns)
                 .HasForeignKey(bc => bc.TicketId);
+
+                model.HasIndex(bc => bc.TicketId).IsUnique();
```
Comment (Ali-YousefiTelori) [inline src/CSharp/EasyMicroservices.SupportsMicroservice.Database/Database/Contexts/SupportContext.cs]: Please remove IsUnique.

**Review by Ali-YousefiTelori — CHANGES_REQUESTED** [review]

**Review by AmirmohammadNemati — APPROVED** [review]

**Review by Ali-YousefiTelori — APPROVED** [review]

**Review by mahdidelzendeh72 — COMMENTED** [review]
thanks

**Pull request merged**